api/controllers/get.js

```diff
@@ -9,7 +9,10 @@ module.exports = function(store) {
 			const user = await store.get(req.params.id);
 			return res.success(user);
 		} catch (e) {
-			return next(e);
+			if (e.path === '_id' && e.name === 'CastError') {
+				return res.failure('Invalid id', 400);
+			}
```

Comment:
Your handling is alright, but it seems hacky because it is added in the wrong place. We want it to be caught by the general error handling stack.

I've invited you to the [respondo](https://github.com/bitkompagniet/respondo) project and assigned you to the [Generic CastError handler](https://github.com/bitkompagniet/respondo/issues/1) issue.